tests/test_be.py

```diff
@@ -1,27 +1,29 @@
-from popgetter.assets import be
-import matplotlib.pyplot as plt
-import geopandas as gpd
+from __future__ import annotations
+
+from pathlib import Path
 
+import geopandas as gpd
+import pytest
 from dagster import (
     build_asset_context,
 )
-import pytest
 
+from popgetter.assets import be
 
 
 @pytest.fixture(scope="module")
 def demo_sectors() -> gpd.GeoDataFrame:
-    return gpd.read_file("popgetter_tests/demo_data/be_demo_sector.geojson")
+    input_path = str(Path(__file__).parent / "demo_data/be_demo_sector.geojson")
```

Comment:
```suggestion
    input_path = str(Path(__file__).parent / "demo_data" / "be_demo_sector.geojson")
```

I like it this way because each path component stands on its own, but this is so minor I wasn't even sure whether to suggest it!